docs/public-networks/how-to/connect/manage-peers.md

```diff
@@ -31,10 +31,11 @@ To reduce the maximum number of peers, use the [`--max-peers`](../../reference/c
 
 The minimum number of peers is set by the `--Xp2p-peer-lower-bound` option, which also has a default of 25. If you reduce the `--max-peers` from the default, you must also set the `--Xp2p-peer-lower-bound` option to the same value or lower. For example, if you decrease `--max-peers` to 20, set `--Xp2p-peer-lower-bound` to 20 or lower.
 
-Note, `Xp2p-peer-lower-bound` is an early access option.
+`--Xp2p-peer-lower-bound` is an early access option.
```

Comment:
Just making this consistent with the other EA options

Reply:
this option has actually been deprecated and will be removed soon https://github.com/hyperledger/besu/pull/6501

Reply:
Thanks @macfarla! I think I will open a separate issue to correct

Reply:
Actually I will just add it to this PR

Reply:
So should I revert this change to just say the option is EA? I think the other section when `--Xp2p-peer-lower-bound` is mentioned is in the options page. Should I not add the deprecation note here and open an issue to fix both instances?

Reply:
Yes, feel free to revert the change here and cover this in a separate PR.